examples/extensions/commands/custom_commands.rst

```diff
@@ -0,0 +1,174 @@
+.. _examples_extensions_custom_commands:
+
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands. These are some of the rules/tips to create custom commands:
+
+* All the custom commands must be located under the ``[YOUR_CONAN_HOME]/extensions/commands/`` folder.
+    * It can be created another folders layer to act like ``scope`` for the commands under it, e.g., ``[YOUR_CONAN_HOME]/extensions/commands/myscope/``.
+* Every custom command file must be named like ``cmd_[YOUR_CUSTOM_COMMAND_NAME].py``.
+* Any command can have sub-commands.
+* It can be performed different formatters to output any command result.
+* It is automatically injected one object instance of :ref:`ConanAPIV2 class<reference_python_api_conan_api_v2>` as first paremeter in the command function.
+
+Let's see some basic examples of custom commands:
+
+Simple "hello" command
+--------------------------
+
+This is the simplest command that runs a ``conan hello`` command and outputs ``Hello World!`` in your console.
+
+First of all, ensure that your ``[YOUR_CONAN_HOME]/extensions/commands/`` folder is already there, otherwise, create it.
+
+Save the ``cmd_hello.py`` file in the mentioned folder. Its content looks like this:
+
+.. code-block:: python
+    :caption: cmd_hello.py
+
+    from conan.api.conan_api import ConanAPIV2
+    from conans.cli.command import conan_command
```

Comment:
why `conans` namespace? isn't V2 supposed to use `conan` namespace?

Reply:
We're maintaining `develop` and `develop2` branches and the conflicts are huge, so, we'll have to keep it till we freeze 1.x and merge `develop2` -> `develop`. After that, we'll update all the imports from docs.